models/user_document.py

```diff
@@ -26,22 +28,23 @@ async def generate_document_name(service_id: int) -> str:
     return f"Заява на {query_result.service_name.lower()}"
 
 
-async def create_user_document_content(*args, **kwargs) -> str:
+async def create_user_document_content(**kwargs) -> str:
     if kwargs.get("service_id") == 1:
-        path_to_template = os.path.join(TEMPLATES_PATH, "hostel_booking_template.docx")
+        path_to_template = Path(TEMPLATES_PATH) / "hostel_booking_template.docx"
```

Comment:
lets put "hostel_booking_template.docx" somewhere to global vars or in file global vars